FV3/conf/configure.fv3.wcoss_cray

```diff
@@ -44,6 +44,10 @@ ifeq ($(GT4PY_DEV),Y)
 CPPDEFS += -DGT4PY_DEV
 endif
 
+ifeq ($(SUBSET_PHYSICS),Y)
```

Comment:
Several copy pastes. Maybe these makefile should include a common file or this behavior should be added to the `./configure` script as a flag...e.g. 
```
./configure --subset-physics <system>
```

Reply:
Including it in the script would also help with documentation, since you could have a help string explaining what the flag does.